epub34/rs/index.html

```diff
@@ -1205,22 +1205,24 @@ <h3>Cascading Style Sheets (CSS)</h3>
 							[=EPUB content documents=].</p>
 					</li>
 					<li>
-						<p id="confreq-css-overrides">SHOULD NOT override the EPUB creator's style sheets, but SHOULD do
-							so in a way that preserves the Cascade when necessary: through a user agent style sheet, or
-							[[html]] [^html-global/style^] attributes.</p>
+						<p id="confreq-css-overrides">MAY override parts of the creator's style sheets (for, e.g., ergonomic or 
+							user interface reasons). If it does, it SHOULD do so in a way that preserves the cascade model
+							of CSS: through a user agent style sheet, or [[html]] [^html-global/style^] attributes.
+							Reading system developers should also publicly document the user agent style sheets and how they
+							interact with EPUB creator's style settings.</p>
 					</li>
 					<li>
-						<p id="confreq-css-user-styles">It MAY override parts of the EPUB creator's style sheet because
-							of user interaction.</p>
+						<p id="confreq-css-user-styles">MAY override parts of the EPUB creator's style sheet as a result
```

Comment:
See above, this is a repetition of a previous sentence I propose to modify. 
Regarding the second sentence, we have kept a requirement above, expressed as "SHOULD apply EPUB creator style sheets as written to EPUB content documents."
Here, it would make sense to write:
"SHOULD provide the user with an affordance that reverts the display to the author's stylesheet". 
This expresses a feature requirement, but does not insist on technical solutions.